src/games/kalambury/board/GameBoard.js

```diff
@@ -21,14 +35,29 @@ const GameBoard = ({
   canChangePhrase,
   ChangePhrase,
   t,
-}) => (
+}) => {
+
+  const [lastSuccess, setLastSuccess] = useState(false);
+  const previousUserGuesses = getUserActions(actions, playerID, "guess")
+
+  useEffect(() => {
+    let guess = previousUserGuesses[0];
+    console.log("Guess", guess, actions, previousUserGuesses, moves)
+    if (!guess) {
+      return;
+    }
+    setLastSuccess(guess.success);
+    return () => setLastSuccess(false); // Do cleanup of state 
+  }, [actions[0].id]);
```

Comment:
@Bombasarkadian WDYT?

Reply:
Does it work then there are no actions? 👀 Won't there be something like `actions[0] is undefined`?

Reply:
yep, it will fail

Reply:
Can we use some dummy object when no action exist? Or is there some cleaner way to do this?
```
const lastUserGuess = getUserActions(actions, playerID, "guess)[0] || { id: null }

useEffect(() => {...}, [lastUserGuess]);
```

Reply:
maybe we can remove `lastSuccess` and this `useEffect` and just use:
```
<Confetti active={lastUserGuess && lastUserGuess[0].success} config={config} className="confetti"/>
```

Reply:
it should work

Reply:
Doesn't this Confetti component require switching from `false` to `true` @turbaszek ? If that's the case it probably won't activate if you successfully guess a few times in a row without any wrong guesses

Reply:
> Can we use some dummy object when no action exist? Or is there some cleaner way to do this?
> 
> ```
> const lastUserGuess = getUserActions(actions, playerID, "guess)[0] || { id: null }
> 
> useEffect(() => {...}, [lastUserGuess]);
> ```

Log is flooding so I suppose it's not best option. 

> maybe we can remove `lastSuccess` and this `useEffect` and just use:
> 
> ```
> <Confetti active={lastUserGuess && lastUserGuess[0].success} config={config} className="confetti"/>
> ```

Doesn't work

Reply:
> Does it work then there are no actions? 👀 Won't there be something like `actions[0] is undefined`?

When there will bo no action?

Reply:
Oh, in the first one I wanted to do `useEffect(() => {...}, [lastUserGuess.id]);` The object does change a lot, but id should stay the same